CuratedContent/psip-podcast.md

```diff
@@ -0,0 +1,32 @@
+# What makes PSIP suitable for Exascale Computing Project?
+
+The Exascale Computing Project focusses on the development of a highly capable exascale ecosystem. With hundreds of people resources and codes of different magnitudes involved in this huge project, what is the path for software sustainabiltiy and lasting software quality?.
```

Comment:
- spelling two s in *focuses*
- if `people, resources  and codes` is a list of three things, missing comma between `people` and `resources`. If `people resources` is a phrase for the kind of resources, I would delete `resources`.
- what does `magnitudes` of codes refer to? Team size, lines of code count? Maybe `sizes` better.